src/components/context/data/processors/jcampProcessor.ts

```diff
@@ -0,0 +1,81 @@
+import { PartialFileList } from 'filelist-utils';
+import { convert } from 'jcampconverter';
+import { v4 } from 'uuid';
+
+import { DataState, MeasurementKind } from '../DataState';
+import { irSpectrumEnhancer } from '../enhancers/irSpectrumEnhancer';
+
+const enhancers: Record<string, typeof irSpectrumEnhancer> = {
+  ir: irSpectrumEnhancer,
+};
+
+export async function jcampProcessor(
```

Comment:
How can I specify that jcampProcessor is of type 'Processor' ?

Reply:
`export const jcampProcessor: Processor = async function (...) {}`